AutomatedTesting/Gem/PythonTests/automatedtesting_shared/base.py

```diff
@@ -96,19 +96,16 @@ def teardown():
         editor_starttime = time.time()
         self.logger.debug("Running automated test")
         testcase_module_filepath = self._get_testcase_module_filepath(testcase_module)
-        pycmd = ["--runpythontest", testcase_module_filepath, f"-pythontestcase={request.node.name}"]
+        pycmd = ["--runpythontest", testcase_module_filepath, f"-pythontestcase={request.node.name}",
+                 "--regset=/Amazon/Preferences/EnablePrefabSystem=true",
```

Comment:
While it is made no longer optional, passing this option is not removed from the framework. I assume this change was intended to make the regset option globally enabled and no longer as easy to optionally disable.